### PR TITLE
Improved relayer learn and build pages

### PR DESCRIPTION
**Pull request by SurgeCode**

### CODE REVIEW

File: docs/1.concepts/abstraction/relayers.md [hunk 1]
```diff
@@ -3,15 +3,17 @@ id: relayers
...
 
 ---
 
 ## How it works
 
-Relayers are a natural consequence of [Meta Transactions](meta-tx.md) ([NEP-366](https://github.com/near/NEPs/blob/master/neps/nep-0366.md)), a special type of transaction which can be best understood as an intent.
+![relayer-overview](/docs/assets/welcome-pages/relayer-overview.png)
 
-The user expresses: _"I want to do a specific action on chain"_ and signs this intent **off-chain**, but does not sends it to the network. Instead, they send the intent to a `Relayer`, which wraps the message into an actual transaction, attaches the necessary funds, and sends it to the network.
+Relayers are a natural consequence of [NEP-366: Meta Transactions](https://github.com/near/NEPs/blob/master/neps/nep-0366.md), a special type of transaction which can be best understood as an intent.
```
Comment (bucanero) [inline docs/1.concepts/abstraction/relayers.md:14]:
```suggestion
Relayers are a natural consequence of [Meta Transactions](meta-tx.md) ([NEP-366](https://github.com/near/NEPs/blob/master/neps/nep-0366.md)), a special type of transaction that can be best understood as an intent.
```

File: docs/2.develop/relayers/welcome.md [hunk 2]
```diff
@@ -1,147 +1,147 @@
...
+import TabItem from '@theme/TabItem';
+import {CodeTabs, Language, Github} from "@site/src/components/codetabs"
 
-This guide will walk you through the steps required to construct a meta transaction relayer using NextJS 13.
+Relayers serve to delegate gas fees to a web service, allowing users to transact on NEAR without the need to acquire the token themselves while still retaining the security of signing their own transactions. This guide will lead you through the components necessary to construct a relayer capable of handling meta transactions.
 
-:::tip
-This guide was originally created by [@microchipgnu](https://x.com/microchipgnu) and can be found in [hackmd](https://hackmd.io/@microchipgnu/meta-transactions-relayer-next-js?utm_source=preview-mode&utm_medium=rec).
-:::
+If you're already aquainted with the technology you can fast track to a [working open source example](https://github.com/SurgeCode/near-relay-example)
```
Comment (bucanero) [inline docs/2.develop/relayers/welcome.md:12]:
```suggestion
If you're already acquainted with the technology, you can fast track to a [working open source example](https://github.com/SurgeCode/near-relay-example) 
```

File: docs/2.develop/relayers/welcome.md [hunk 3]
```diff
@@ -1,147 +1,147 @@
...
+import TabItem from '@theme/TabItem';
+import {CodeTabs, Language, Github} from "@site/src/components/codetabs"
 
-This guide will walk you through the steps required to construct a meta transaction relayer using NextJS 13.
+Relayers serve to delegate gas fees to a web service, allowing users to transact on NEAR without the need to acquire the token themselves while still retaining the security of signing their own transactions. This guide will lead you through the components necessary to construct a relayer capable of handling meta transactions.
 
-:::tip
-This guide was originally created by [@microchipgnu](https://x.com/microchipgnu) and can be found in [hackmd](https://hackmd.io/@microchipgnu/meta-transactions-relayer-next-js?utm_source=preview-mode&utm_medium=rec).
-:::
+If you're already acquainted with the technology, you can fast track to a [working open source example](https://github.com/SurgeCode/near-relay-example)
```
Comment (bucanero) [inline docs/2.develop/relayers/welcome.md:12]:
```suggestion
:::tip

If you're already acquainted with the technology, you can fast track to a [working open source example](https://github.com/SurgeCode/near-relay-example) 

:::
```

**Review by bucanero — CHANGES_REQUESTED** [review]
hello @SurgeCode , thanks for the PR.

It seems that some images that you are linking are missing:

```
/docs/assets/welcome-pages/relayer-overview.png
/docs/assets/welcome-pages/relayer-overview-technical.png
```

can you add the missing files, or remove the references? thanks!

**Comment by SurgeCode** [comment]
> hello @SurgeCode , thanks for the PR.
> 
> It seems that some images that you are linking are missing:
> 
> ```
> /docs/assets/welcome-pages/relayer-overview.png
> /docs/assets/welcome-pages/relayer-overview-technical.png
> ```
> 
> can you add the missing files, or remove the references? thanks!

Thanks for the heads up, added them

**Review by bucanero — APPROVED** [review]

**Pull request merged**